packages/plugins/plugin-markdown/src/extensions.tsx

```diff
@@ -155,25 +206,31 @@ const onRenderLink = (onSelectObject: (id: string) => void) => (el: Element, url
         target: '_blank',
       };
 
-  const LinkIcon: Icon = isInternal ? ArrowSquareDown : ArrowSquareOut;
-
-  createRoot(el).render(
-    <StrictMode>
-      <a {...options} className={hover}>
-        <LinkIcon weight='bold' className={mx(getSize(4), 'inline-block leading-none mis-1 cursor-pointer')} />
-      </a>
-    </StrictMode>,
+  renderRoot(
+    el,
+    <a {...options} className={style.hover}>
+      <Icon
+        icon={isInternal ? 'ph--arrow-square-down--bold' : 'ph--arrow-square-out--bold'}
+        size={4}
+        classNames={style.icon}
+      />
+    </a>,
   );
 };
 
 const renderLinkTooltip = (el: Element, url: string) => {
   const web = new URL(url);
-  createRoot(el).render(
-    <StrictMode>
```

Comment:
no more strict mode for these?

Reply:
temporary; these will be replaced by non-react DOM directly.